bin/propolis-server/src/lib/stats.rs

```diff
@@ -132,33 +131,32 @@ impl Producer for ServerStatsOuter {
 /// This method attempts to register a _single time_ with Nexus. Callers should
 /// arrange for this to be called continuously if desired, such as with a
 /// backoff policy.
-pub async fn start_oximeter_server(
+///
+/// The returned server will attempt to register with Nexus in a background
+/// task, and will periodically renew that registration. The returned server is
+/// running, and need not be poked or renewed to successfully serve metric data.
+pub fn start_oximeter_server(
     id: Uuid,
     config: &MetricsEndpointConfig,
     log: &Logger,
     registry: &ProducerRegistry,
 ) -> Result<Server, Error> {
     // Request an ephemeral port on which to serve metrics.
-    let my_address = SocketAddr::new(config.propolis_addr.ip(), 0);
+    let producer_address = SocketAddr::new(config.propolis_addr.ip(), 0);
     let registration_address = config.metric_addr;
     info!(
         log,
         "Attempting to register with Nexus as a metric producer";
-        "my_address" => %my_address,
+        "producer_address" => %producer_address,
         "nexus_address" => %registration_address,
     );
 
-    let dropshot_config = ConfigDropshot {
-        bind_address: my_address,
-        request_body_max_bytes: 2048,
-        default_handler_task_mode: HandlerTaskMode::Detached,
-    };
-
     let server_info = ProducerEndpoint {
         id,
         kind: ProducerKind::Instance,
-        address: my_address,
-        base_route: "/collect".to_string(),
+        address: producer_address,
+        // NOTE: This field is unused, and will be removed in the future.
+        base_route: String::new(),
```

Comment:
tiny nit, not important: is there an Oximeter issue or documentation or something that this comment could include a link to?

Reply:
Just made https://github.com/oxidecomputer/omicron/issues/5658 to track, will link here.

Reply:
lovely, thank you!